### ISSUE TITLE
Questions about training procedure

### ISSUE THREAD

**Issue by BrianWayland** [Uehwan/SimVODIS#5]
@Uehwan Thank you for sharing your implementation kindly. When learning your code I find two problems:
- in train.py, the dataset is didn't looped through and just used the first batch for training, as shown in follows:
 ```python
    ds = iter(trainer.train_loader)
    inputs = next(ds)
```
- after predict depth and pose, the loss is not calculated to back propagate gradient and optimize parameters.
